json/src/fs2/data/json/JsonException.scala

```diff
@@ -15,4 +15,4 @@
  */
 package fs2.data.json
 
-class JsonException(msg: String) extends Exception(msg)
+case class JsonException(msg: String) extends Exception(msg)
```

Comment:
Could be handy to know which fields were missing without having to match the message, wdyt?

Reply:
Could be, I was too lazy too add a specific exception but since you are asking :)